src/components/Subtitle/SubtitleApp.scss

```
@@ -36,25 +36,127 @@
+  }
+
+  &__hint {
+    font-size: 12px;
+    color: rgba(255, 255, 255, 0.45);
+  }
+
+  // One line, always. The window can be resized to any height, so the error
+  // must never push the buttons out of view.
+  &__error {
+    font-size: 12.5px;
+    // The app's error-text-on-dark colour (MainPanel.scss:150,156).
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)